docs/architecture/adr/0027-angular-i18n-migration.md

```diff
@@ -0,0 +1,185 @@
+---
+adr: "0027"
+status: Proposed
+date: 2025-08-01
+tags: [clients, angular, i18n]
+---
+
+# 0027 - Improved Localization
+
+<AdrTable frontMatter={frontMatter}></AdrTable>
+
+## Context and Problem Statement
+
+We currently use a homegrown localization system based on the
+https://developer.chrome.com/docs/extensions/reference/api/i18n api. This has served us reasonably
+well but comes with many limitations which we've had to workaround over the years.
+
+Most notable it lacks any native capability for interpolation which is useful for adding markdown
+such as marking words as italic or bold or inserting links into strings. In most scenarios we've
+handled this by splitting the sentence into multiple localized strings but that is not always
+optimal in all locales.
+
+Another limitation is pluralization where we currently can't distinguish phrases that are singular
+or plural which we've often mitigated by adding (s) at the end which again provides a sub-optimal
+experience as different languages handles names, numbers and dates differently.
+
+In order to better handle more complex localization scenarios we'd like to explore changing the
+underlying localization engine.
+
+## Considered Options
+
+- **Keep chrome.i18n** - Continue using our current homegrown localization system
+- **Continue building our own** - Expand our custom localization framework
+- **Angular Localization (@angular/localize)** - Use Angular's built-in i18n framework
+- **ngx-translate** - Use the popular Angular translation library
+- **`Transloco`** - Use the modern Angular translation library
+- **angular-i18next** - Use i18next with Angular integration
+
+### Keep `chrome.i18n`
+
+**Pros:**
```

Comment:
🎨 I would prefer using `####` for these and dropping the colon.

Reply:
I dropped the colon but didn't make them titles since they would show up in the sidebar which would just be noisy.